### ISSUE TITLE
Very different accuracy for the "3 - Neural Networks" MNIST examples

### ISSUE THREAD

**Issue by ollop** [aymericdamien/TensorFlow-Examples#250]
Hi,

I looked at the "Simple Neural Network (eager api)" and the "Simple Neural Network (tf.layers/estimator api)". The resulting accuracy is very different.

Reasons for the differences are as follows. The estimator example has a no activation function, while eager example uses ReLU. estimator uses gradient descent, eager Adam. Learning rates are also different (which might make sense when different optimizers and activation functions are used).

I found the differences rather confusing as I wondered where the activation function is specified until I realized that none is specified which is probably not what beginners want to use.

Would be great if the examples would show the same behavior.
